README.md

```diff
@@ -1,18 +1,28 @@
 # Installation
 ## Locally
 * postgres server installed and running
-	* `brew install postgres`
-	* `pg_ctl -D /usr/local/var/postgres start`
-* heroku cli installed
-	* `brew install heroku`
+    * `brew install postgres`
+    * `pg_ctl start -D /usr/local/var/postgres`
```

Comment:
Did the previous version not work on your machine?

Reply:
nope

```
mbp-IT002597:~ ethomas$ pg_ctl --version
pg_ctl (PostgreSQL) 10.3
mbp-IT002597:~ ethomas$ pg_ctl --help | head -10
pg_ctl is a utility to initialize, start, stop, or control a PostgreSQL server.

Usage:
  pg_ctl init[db] [-D DATADIR] [-s] [-o OPTIONS]
  pg_ctl start    [-D DATADIR] [-l FILENAME] [-W] [-t SECS] [-s]
                  [-o OPTIONS] [-p PATH] [-c]
  pg_ctl stop     [-D DATADIR] [-m SHUTDOWN-MODE] [-W] [-t SECS] [-s]
  pg_ctl restart  [-D DATADIR] [-m SHUTDOWN-MODE] [-W] [-t SECS] [-s]
                  [-o OPTIONS] [-c]
  pg_ctl reload   [-D DATADIR] [-s]
```

I tried checking the pg_ctl release notes to see if something changed. Couldn't find anything. But putting the optional args before `start` resulted in an error for me